www/src/index.tsx

```diff
@@ -1,12 +1,19 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import { BrowserRouter as Router } from 'react-router-dom';
+import Auth0ProviderWithHistory from './providers/Auth0ProviderWithHistory';
+
 import './index.css';
 import App from './App';
 import reportWebVitals from './reportWebVitals';
 
 ReactDOM.render(
     <React.StrictMode>
-        <App />
+        <Router>
```

Comment:
`Auth0Provider` uses a `Redirect` component that needs a `BrowserRouter` as its parent.